Message Queue documentation for UXPROD-340

mreno-EBSCO opened the pull request:
Added some basic Kafka/Zookeeper documentation for mod-aes usage as part of UXPROD-340.

mreno-EBSCO committed:
Message Queue documentation for UXPROD-340

Added some basic Kafka/Zookeeper documentation for mod-aes usage as part of UXPROD-340.

hjiebsco commented on README.md:
```diff
@@ -13,6 +13,57 @@ The mod-aes module provides FOLIO asynchronous event service (AES)
+
+Execute the container as follows:
+- Start: `$ /usr/local/bin/docker-compose up -d`
+- Stop: `/usr/local/bin/docker-compose stop`
```
Start has $ but not Stop. Make it consistent?

mreno-EBSCO committed:
Cleanup based on review comments

- Removed unneeded ports (only needed for ensemble execution).
- Added the `$` prompt missing from a shell command.

hjiebsco reviewed (approved):
Tried the configuration and it works for me.
